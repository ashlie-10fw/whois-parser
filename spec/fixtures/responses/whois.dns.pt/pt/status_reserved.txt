Domain: wiki.pt
Domain Status: Reserved
Creation Date: 07/05/2015 23:40:09
Expiration Date: 06/05/2026 23:59:09
Owner Name: 
Owner Address: 
Owner Locality: 
Owner ZipCode: 
Owner Locality ZipCode: 
Owner Country Code: 
Owner Email: 
Admin Name: SOOMA.COM - WEB SERVICES, LDA
Admin Address: Rua Dom Jo√£o I  99
Admin Locality: Porto
Admin ZipCode: 4450-164
Admin Locality ZipCode: Matosinhos
Admin Country Code: PT
Admin Email: gestao.dominios@sooma.com,daf@sooma.com
