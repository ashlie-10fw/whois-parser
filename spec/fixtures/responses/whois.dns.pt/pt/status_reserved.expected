#status
  %s == :reserved

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2015-05-07 23:40:09 UTC}

#updated_on
  %s %ERROR{AttributeNotSupported}

#expires_on
  %s %CLASS{time}
  %s %TIME{2026-05-06 23:59:09 UTC}

#nameservers
  %s %CLASS{array}
  %s == []
